File: mvpLibrary/src/main/java/cn/cbsd/mvplibrary/net/SoapProvider.java
package cn.cbsd.mvplibrary.net;

import me.jessyan.rxerrorhandler.core.RxErrorHandler;

/**
 * 当前类注释:
 * Author: zhenyanjun
 * Date  : 2018/1/19 11:11
 */

public interface SoapProvider {

    RxErrorHandler configHandleError(boolean isNeedShowToast);
}

File: mvpLibrary/src/main/java/cn/cbsd/mvplibrary/net/SoapProvider.kt
package cn.cbsd.mvplibrary.net

import me.jessyan.rxerrorhandler.core.RxErrorHandler

/**
 * 当前类注释:
 * Author: zhenyanjun
 * Date  : 2018/1/19 11:11
 */

interface SoapProvider {

    fun configHandleError(isNeedShowToast: Boolean): RxErrorHandler
}
